shell/platform/windows/flutter_window.h

```diff
@@ -9,161 +9,311 @@
 #include <vector>
 
 #include "flutter/fml/macros.h"
+#include "flutter/shell/platform/common/alert_platform_node_delegate.h"
 #include "flutter/shell/platform/common/geometry.h"
 #include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/windows/direct_manipulation.h"
 #include "flutter/shell/platform/windows/flutter_windows_view.h"
-#include "flutter/shell/platform/windows/window.h"
+#include "flutter/shell/platform/windows/keyboard_manager.h"
+#include "flutter/shell/platform/windows/sequential_id_generator.h"
+#include "flutter/shell/platform/windows/text_input_manager.h"
 #include "flutter/shell/platform/windows/window_binding_handler.h"
+#include "flutter/shell/platform/windows/windows_lifecycle_manager.h"
+#include "flutter/shell/platform/windows/windows_proc_table.h"
+#include "flutter/shell/platform/windows/windowsx_shim.h"
+#include "flutter/third_party/accessibility/ax/platform/ax_fragment_root_delegate_win.h"
+#include "flutter/third_party/accessibility/ax/platform/ax_fragment_root_win.h"
+#include "flutter/third_party/accessibility/ax/platform/ax_platform_node_win.h"
+#include "flutter/third_party/accessibility/gfx/native_widget_types.h"
 
 namespace flutter {
 
 // A win32 flutter child window used as implementations for flutter view.  In
 // the future, there will likely be a CoreWindow-based FlutterWindow as well.
 // At the point may make sense to dependency inject the native window rather
 // than inherit.
-class FlutterWindow : public Window, public WindowBindingHandler {
+class FlutterWindow : public KeyboardManager::WindowDelegate,
```

Comment:
nit: in terms of naming, I'd be tempted to go with `Window` and rely on the namespace to clarify that this is a `flutter::Window` rather than something more specific, and go with filenames `window.{h,cc}` for the same reasons.

That said, that could be a separate change. There are a few other places where the `Flutter` prefix remains on things that don't particularly need it and we could do them all in one go.

Reply:
That is somewhat along the lines of what I was thinking, as we still also have `FlutterWindowsEngine`, `FlutterWindowsView`, etc..

Reply:
Sounds good! Let's leave that for a separate cleanup then.